internal/ethereum/deploy_contract_prepare_test.go

```diff
@@ -39,19 +40,91 @@ const samplePrepareDeployTX = `{
 	"gas": 1000000,
 	"nonce": "111",
 	"value": "12345678901234567890123456789",
-	"contract": "0xfeedbeef",
+	"contract": "0xdeadbeef",
```

Comment:
🤣

Reply:
Yeah a slightly cheeky way to ensure the asserts in the tests don't match on the contract bytes when checking for integers that match `0xdeadbeef`